project/src/components/Authorization/components/logout.tsx

```diff
@@ -0,0 +1,26 @@
+import { useAppDispatch } from '../../../hooks';
+import { fetchLogoutAction} from '../../../store/api-action';
+import '../style.scss';
+
+export const Logout = () => {
+  const dispatch = useAppDispatch();
+
+  const signOutHandler = (evt : React.MouseEvent) => {
```

Comment:
```suggestion
  const handleSignOutButtonClick = (evt : React.MouseEvent<HTMLButtonElement>) => {
```

Use consistent naming <img width="720" alt="image" src="https://github.com/SamoylenkVi/wtw_cinema/assets/47995886/4d56cc01-2013-4d71-843a-c57802b0e676">

Reply:
fixed